scratchai/one_call.py

```diff
@@ -38,16 +40,26 @@ def classify(path:str):
   a resnet output the predicted value.
```

Comment:
Add a note that if `trf is None` then it defaults to `rz256_cc224_tt_normimgnet`